cmd/server/pipeline_company_name_cleanup.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 The Moov Authors
+// Use of this source code is governed by an Apache License
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"strings"
+)
+
+type companyNameCleanupStep struct {
+}
+
+func (s *companyNameCleanupStep) apply(in *Name) error {
+	switch {
+	case in.sdn != nil && in.sdn.SDNType == "":
+		in.Processed = removeCompanyTitles(in.Processed)
+
+	case in.ssi != nil && in.ssi.Type == "":
+		in.Processed = removeCompanyTitles(in.Processed)
+	}
+	return nil
+}
+
+// original list: inc, incorporated, llc, llp, co, ltd, limited, sa de cv, corporation, corp, ltda,
+//                open joint stock company, pty ltd, public limited company, ag, cjsc, plc, as, aps,
+//                oy, sa, gmbh, se, pvt ltd, sp zoo, ooo, sl, pjsc, jsc, bv, pt, tbk
+
+var (
+	companySuffixReplacer = strings.NewReplacer(
+		" CO.", "",
```

Comment:
Is this going to be run _after_ punctuation has already been removed? If so, we need to match against `" CO"`, rather than `" CO."`, etc.

I was also thinking it would probably be good to run this _before_ removing stopwords. That way we can also use this to remove identifiers like "SA de CV".

Reply:
Those are good points. I wrote up a quick test covering the entire default pipeline to help us verify the steps.

I agree we should remove company suffixes before stopwords and changed the ordering. 

https://github.com/moov-io/watchman/pull/217/commits/d1aac492994c400c3ca3735665a1f31a93ad8701